torch/_torch_docs.py

```diff
@@ -5444,6 +5444,36 @@ def merge_dicts(*dicts):
     tensor([ 0.7153,  0.7481,  0.2920,  0.1458])
 """.format(**common_args))
 
+add_docstr(torch.logit,
+           r"""
+logit(input, eps=None, out=None) -> Tensor
+
+Returns a new tensor with the logit of the elements of :attr:`input`.
+:attr:`input` is clamped to [eps, 1 - eps] when eps is not None.
```

Comment:
Can you please note that when eps is None then logit(p) for p<0 or p>1 yields nan.

Reply:
Done.